### DISCUSSION TITLE
How to embed form as an iframe

### DISCUSSION THREAD

**Discussion by terryMarsault** [OpnForm/OpnForm#123] [Q&A] [answered]
Hi,
I've created my form using the official managed service in the Cloud and now I could like to embed this form as an iframe into my website (or any other solution inside my website). How can I acheive this?
Thanks!

**Comment by JhumanJ** [comment, marked as answer]
Hey @terryMarsault there's an iframe code provided when you click the "Embed form" option.

**Comment by terryMarsault** [comment]
Hi @JhumanJ, thanks, I found it in the admin panel: 
![image](https://github.com/JhumanJ/OpnForm/assets/132718573/b193143e-894a-4e1b-8d9d-4b1fd4315bad)
I think it should be noted somewhere in the help page? (which is, BTW, not accessible from the hosted version)

**Comment by JhumanJ** [comment]
Great that you found it :) the docs are not available anymore, because Crisp, the tool we use for it is paid. We’ll try find a way to monetise the project and improve documentation, support etc.
